tests/modules/mcuboot/key-locking/sysbuild/mcuboot.conf

```diff
@@ -0,0 +1,11 @@
+#
+# Copyright (c) 2025 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+CONFIG_PM_PARTITION_SIZE_MCUBOOT=0x10000
+CONFIG_FPROTECT=n
+# Test will access only the first slot, the 226.
+CONFIG_BOOT_SIGNATURE_KMU_SLOTS=3
```

Comment:
in the comments is that only the first slot is used, and 3 is set in config, is it ok? do we need to set number of KMU slots?

Reply:
We only need one for the test. I was considering to run a loop on all three but decided to not do so.
I was wandering whether I should run both the =3 and =1 values to check in change in Key count affects key locks.
Probably should remove this from conf and move it to the testcase.yaml.